notebook/scale-test.py

```diff
@@ -130,6 +130,7 @@ def get_cookie(session, name):
 
     print(f'successes: {len(times)} / {n} = {len(times) / n}')
     print(f'mean time: {sum(times) / n}')
+    print(f'quantiles 50%/95%/99%:: {np.quantile(times, [.5, .95, .99])}')
     print(f'max time: {max(times)}')
```

Comment:
Want to include max with the quantiles?  50/95/99/max?

Reply:
good idea, done